app/views/components/input_radio_with_hint.scala.html

```diff
@@ -0,0 +1,67 @@
+@*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import viewmodels.RadioOption
+
+@(
+    field: Field,
+    legend: String,
+    legendClass: Option[String] = None,
+    hint: Option[String] = None,
+    trackGa: Boolean = false,
+    headingIsLegend: Boolean = false,
+    inputs: Seq[(RadioOption, String)]
+)(implicit messages: Messages)
+
+<div class="form-group @if(field.hasErrors){form-group-error}">
+    <fieldset id="@{field.id}"
+        @if(hint.nonEmpty || field.hasErrors) {
+            aria-describedby="@if(hint.nonEmpty){hint-@{field.id}}@if(hint.nonEmpty && field.hasErrors){ }@if(field.hasErrors){error-message-@{field.id}-input}"
+        }
+    >
+
+        @if(headingIsLegend){
+            <legend>
+                <h1 @if(legendClass.nonEmpty){class="@legendClass" }else{class="heading-large"}>@legend</h1>
+            </legend>
+        } else {
+            <legend @if(legendClass.nonEmpty){class="@legendClass" }>
+                @legend
+            </legend>
+        }
+
+        @if(hint.nonEmpty){
+            <div class="form-hint" id="hint-@{field.id}">@hint</div>
+        }
+        @field.errors.map { error =>
+            <span class="error-message" id="error-message-@field.id-input">
+                <span class="visually-hidden">@messages("site.error") </span>@messages(error.message, error.args: _*)
+            </span>
+        }
+        @for(input <- inputs) {
+            <div class="multiple-choice">
+                <input id="@{input._1.id}" type="radio" name="@{field.id}" value="@{input._1.value}" aria-describedby="@{input._1.id}.hint" @if(field.value == Some(input._1.value)){checked="checked"} @if(trackGa){data-journey-click="maintain-a-trust-frontend:click:@{input._1.id}"} />
```

Comment:
Does `maintain-a-trust-frontend` need to be changed to `trusts-frontend` for GA tracking?

Reply:
fixed now